src/components/home/Directions.jsx

```diff
@@ -1,5 +1,16 @@
+import React from "react";
+
 const Directions = () => {
-  return <div>Directions</div>;
+  return (
+    <div className=" flex flex-col justify-center items-center bg-[#83694D] h-screen">
+      <div className="text-white font-extrabold text-6xl ">
+        How to get to Our Garden
+      </div>
+      <div className=" h-[70%] w-[90%] bg-[#C0BDA5] mt-[3%] mb-[3%] border-2 border-[#3E2913] rounded-lg">
```

Comment:
don't do a top and bottom margin separately, do it at the same time with my-[3%]